vms/platformvm/state/state.go

```diff
@@ -292,9 +305,17 @@ type state struct {
 	currentSupply, persistedCurrentSupply uint64
 	// [lastAccepted] is the most recently accepted block.
 	lastAccepted, persistedLastAccepted ids.ID
+	indexedHeights                      *heightRange
 	singletonDB                         database.Database
 }
 
+// heightRange is used to track which heights are safe to use the native DB
```

Comment:
q: so this will go when we're sure validators have all switched to the native index right?